main_COPY_AND_RENAME.py

```diff
@@ -29,7 +37,80 @@
         if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
             pygame.quit()
             sys.exit()
-        
+        # /--------------------------------/
+        # The first way of interacting with events
+        """"
+        elif event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_RIGHT and x < screen_width - width:
+                rect_x += rect_speed
+            elif event.key == pygame.K_LEFT and x > 0:
+                rect_x -= rect_speed
+            elif event.key == pygame.K_UP and y > 0:
+                rect_y -= rect_speed
+            elif event.key == pygame.K_DOWN and y < screen_height - height:
+                rect_y += rect_speed
+        elif event.type == pygame.MOUSEBUTTONDOWN:
+            if event.button == 1:
+                print("You pressed the left mouse botton!")
+                color = RED # Change colour to red
+            elif event.button == 3:
+                print("You pressed the right mouse button!")
+                color = GREEN # Change colour to green
+            elif event.button == 2:
+                # add your own code
+                print("You pressed the mouse wheel button!")
+            elif event.buttom == 4:
+                # add your own code
+                print("You used the mouse wheel scroll up!")
+            elif event.buttom == 5:
+                # add your own code
+                print("You used the mouse wheel scroll down!")
+            """
+
+    # /--------------------------------/
+    # The second way of interacting with events
+
+    keys = pygame.key.get_pressed()
+    if keys[pygame.K_UP] and rect_y > 0:
+        # y > 0 so we can't go out of our screen, try getting
+        # rid of it and see what happens if you go up
+        rect_y -= rect_speed # Going up
+    elif keys[pygame.K_LEFT] and rect_x > 0:
+        # x > 0 so we can't go out of our screen
+        rect_x -= rect_speed # Going left
+    elif keys[pygame.K_RIGHT] and rect_x < SCREEN_WIDTH - rect_width:
+        # x < screen_width - width so we can't go out of our screen
+        rect_x += rect_speed # Going right
+    elif keys[pygame.K_DOWN] and rect_y < SCREEN_HEIGHT - rect_height:
+        # y < screen_height - height so we can't go out of our screen
+        rect_y += rect_speed # Going down
+
+    # /-----------------------------/
+    # Mouse events
+    mouses = pygame.mouse.get_pressed()
+    # runs while you hold it
+    if(mouses[0]): # If left mouse pressed
```

Comment:
PEP8 - don't add brackets around `if` statements unnecessarily and put two spaces between the end of your line and the start of a comment

```suggestion
    if mouses[0]:  # If left mouse pressed
```